scripts/make_zephyr_sdk.sh

```diff
@@ -129,6 +130,13 @@ if [ -n "$file_gcc_riscv32" ]; then
   echo "echo \"\"" >>$setup
 fi
 
+if [ -n "$file_gcc_x86_64" ]; then
+  echo "tar -C \$target_sdk_dir -jxf ./$file_gcc_x86_64 > /dev/null &" >> $setup
```

Comment:
`printf` would make quoting clearer and more robust, something like:
```
printf 'tar -C "%s" -jxf "%s" >/dev/null &'   '${target_sdk_dir}'  "./${file_gcc_x86_64}" >> "$setup"
```

Reply:
Alternatively, a here-document would avoid the repetition:
```
cat <<EOFSETUP > "$setup"
tar -C "\${target_sdk_dir}" ...
etc.
EOFSETUP


```
Either would be better.

Reply:
Happy to take that as a followup cleanup patch